### PR TITLE
Added Cobertura formatter

### PR DESCRIPTION
**Pull request by jessebs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -588,6 +588,11 @@ JSON formatter for SimpleCov
 
 A formatter that generates a coverage badge for use in your project's readme using ImageMagick.
 
+### [simplecov-cobertura](https://github.com/dashingrocket/simplecov-cobertura)
+*by Jesse Bowes*
```
Comment (bf4) [inline README.md]: Please add another `#`. i.e. `#### [simplecov-cobertura](https://github.com/dashingrocket/simplecov-cobertura)
*by Jesse Bowes*`

and amend commit

**Comment by bf4** [comment]
Thanks

**Comment by jessebs** [comment]
Fixed.  Build failed but appears to be unrelated to my change.

**Comment by bf4** [comment]
I rebase the commits for you.

**Comment by jessebs** [comment]
Thanks.  I guess I messed up the rebase.

**Pull request closed without merging**